src/Sensors/Sentinel1/Sentinel1Types/MetaDataTypes.jl

```diff
@@ -184,4 +185,69 @@ Base.@kwdef struct Sentinel1MetaData <: MetaData
 end
 
 
-get_polarisation(meta_data::Sentinel1MetaData) = meta_data.header.polarisation
\ No newline at end of file
+get_polarisation(meta_data::Sentinel1MetaData) = meta_data.header.polarisation
+get_range_sampling_rate(meta_data::Sentinel1MetaData) = meta_data.product.range_sampling_rate
+get_azimuth_frequency(meta_data::Sentinel1MetaData) = meta_data.image.azimuth_frequency
+get_slant_range_time_seconds(meta_data::Sentinel1MetaData) = meta_data.image.slant_range_time_seconds
+get_time_range(meta_data::Sentinel1MetaData) = (meta_data.header.start_time, meta_data.header.stop_time)
+get_reference_time(meta_data::Sentinel1MetaData) = meta_data.reference_time
+
+function get_burst_start_times(meta_data::Sentinel1MetaData)
+    return [element.azimuth_time for element in meta_data.bursts]
+end
```

Comment:
maybe it's more a question to be answered when we talk next time: but I am not sure why these functions are in here. I suppose I expected this MetaDataTypes.jl file to only contain stucts / type definitions. You can explain to me when we talk next time :)

Reply:
You are probably right. It is just a habit of object-oriented programming where the class and class methods are in the same file. 

Let's leave it here. I do not see any better places to put this without refactoring the folder structure again and we do not have time for that. It is just getter functions with minimal logic

Reply:
I agree :)